.gitignore

```diff
@@ -6,4 +6,4 @@ results/
 testing/
 testing*
 *.pyc
-.nf-test*
```

Comment:
This should be in the gitignore

Reply:
`nf-core pipelines lint --fix files_unchanged` says not

Reply:
Well it's wrong unless you want to commit your nf-test cache folder ;)